v1/lib/static/css/main.css

```diff
@@ -2220,14 +2220,14 @@ input::placeholder {
 }
 
 .showcaseSection .logos img {
-  max-height: 128px;
+  height: 128px;
   padding: 20px;
   width: 128px;
 }
 
 @media only screen and (max-width: 735px) {
   .showcaseSection .logos img {
-    max-height: 64px;
+    height: 64px;
```

Comment:
Same here.

Reply:
@yangshun Updated, waited for the deploy env to be built - but it is on hold for some reason